buzzmobile/tests/simulation/test_gazebo_connection.py

```diff
@@ -1,14 +1,17 @@
-from tests.test_utils import RosTest, with_launch_file, check_topic
+from pyrostest import RosTest, with_launch_file
 from rosgraph_msgs.msg import Clock
+import time
 
 class TestGazeboConnection(RosTest):
     """Tests for simulation testing infrastructure."""
 
     @with_launch_file('buzzmobile', 'simulation.launch')
     def test_clock_runs(self):
         """Test that simulation.launch correctly launches gazebo"""
-        with check_topic('/clock', Clock) as ct:
+        with self.check_topic('/clock', Clock) as ct:
             assert(ct.message)
-        with check_topic('/clock', Clock) as ct:
-            assert(ct.message)
-
+            print(ct.message)
+            time.sleep(30)
```

Comment:
Try moving this outside of the with block and see what happens/if you can shorten it.